code/game/objects/machinery/nuclearbomb.dm

```diff
@@ -249,6 +249,7 @@
 
 ///Toggles the timer on or off
 /obj/machinery/nuclearbomb/proc/toggle_timer(mob/user)
+	var/area/area = get_area(src)
 	if(exploded)
```

Comment:
you should declare this just before you use it

Reply:
Done!